sites/public/src/ListingView.tsx

```diff
@@ -0,0 +1,400 @@
+                    unitSummaries={listing.property.unitsSummarized.byUnitType}
+                    disableAccordion={listing.disableUnitsAccordion}
+                  />
+                }
+              />
+            </dl>
+            <AdditionalFees listing={listing} />
+          </div>
+        </ListingDetailItem>
+
+        {listing.property?.buildingAddress.latitude && listing.property?.buildingAddress.longitude && (
+          <ListingDetailItem
+            imageAlt={t("listings.neighborhoodBuildings")}
+            imageSrc="/images/listing-neighborhood.svg"
+            title={t("listings.sections.neighborhoodTitle")}
+            subtitle={t("listings.sections.neighborhoodSubtitle")}
+            desktopClass="bg-primary-lighter"
+          >
+            <div className="listing-detail-panel">
+              <ListingMap address={listing.property.buildingAddress} listing={listing} />
+            </div>
+          </ListingDetailItem>
+        )}
+
+        {(listing.requiredDocuments || listing.programRules || listing.specialNotes) && (
+          <ListingDetailItem
+            imageAlt={t("listings.additionalInformationEnvelope")}
+            imageSrc="/images/listing-legal.svg"
+            title={t("listings.sections.additionalInformationTitle")}
+            subtitle={t("listings.sections.additionalInformationSubtitle")}
+          >
+            <div className="listing-detail-panel">
+              {listing.requiredDocuments && (
+                <div className="info-card">
+                  <h3 className="text-serif-lg">{t("listings.requiredDocuments")}</h3>
+                  <p className="text-sm text-gray-700">
+                    <Markdown
+                      children={listing.requiredDocuments}
+                      options={{ disableParsingRawHTML: true }}
+                    />
+                  </p>
+                </div>
+              )}
+              {listing.programRules && (
+                <div className="info-card">
+                  <h3 className="text-serif-lg">{t("listings.importantProgramRules")}</h3>
+                  <p className="text-sm text-gray-700">
+                    <Markdown
+                      children={listing.programRules}
+                      options={{ disableParsingRawHTML: true }}
+                    />
+                  </p>
+                </div>
+              )}
+              {listing.specialNotes && (
+                <div className="info-card">
+                  <h3 className="text-serif-lg">{t("listings.specialNotes")}</h3>
+                  <p className="text-sm text-gray-700">
+                    <Markdown
+                      children={listing.specialNotes}
+                      options={{ disableParsingRawHTML: true }}
+                    />
+                  </p>
+                </div>
+              )}
+            </div>
+          </ListingDetailItem>
+        )}
+      </ListingDetails>
+    </article>
+  )
+}
```

Comment:
This feels like it could be broken up into smaller components to be used by this. Not saying we need to do that now, per se, there's just a lot going on in this file.

Reply:
Agreed.